Makefile

```diff
@@ -0,0 +1,21 @@
+all:
+	npm run build
```

Comment:
Do you want to just use npm?  I'm ok with that since it turns out yarn, while a drop in replacement for npm with respect to module management and package.json compliancy, does have a slightly different CLI usage (ie yarn install isn't the same as npm install...that's yarn add).